testsuite/tests/tool-ocamlobjinfo/Makefile

```diff
@@ -0,0 +1,47 @@
+BASEDIR=../..
+
+LD_PATH=
+
+.PHONY: default
+default:
+	@if ! $(SUPPORTS_SHARED_LIBRARIES) || $(BYTECODE_ONLY) ; then \
+	  echo 'skipped (.cmxs not built)'; \
+	elif ! grep -q HAS_LIBBFD $(TOPDIR)/config/s.h ; then \
+	  echo 'skipped (BFD library not available)'; \
+	else \
+	  case " $(OTHERLIBRARIES) " in \
+	    *' unix '*) \
+	       $(SET_LD_PATH) OCAMLLIB=$(TOPDIR)/tools $(MAKE) run;; \
+	    *' win32unix '*) \
+	       $(SET_LD_PATH) OCAMLLIB=$(TOPDIR)/tools $(MAKE) run;; \
```

Comment:
The two cases are the same, maybe they could be factorized with a `|` in the patterns?